sklearn/polynomial_chaos/_pcr.py

```diff
@@ -0,0 +1,533 @@
+        X_scaled = np.zeros_like(X)
+        for j, (distribution, polynomial) in enumerate(
+            zip(self.distributions_, self.polynomials_)
+        ):
+            X_scaled[:, j] = polynomial.scale_features_from_distribution(
+                X[:, j], distribution
+            )
+
+        # scale outputs
+        self.output_mean_ = np.mean(y) if self.scale_outputs else 0
+        self.output_std_ = np.std(y) if self.scale_outputs else 1
+        y = y - self.output_mean_
+        y = y / self.output_std_
+
+        self.multiindices_ = self.multiindices
+        self.strategy_ = BasisIncrementStrategy.from_string(strategy)
+
+        # adaptive basis growth
+        for iter in range(max_iter):
+            # create orthogonal polynomial basis transformer
+            basis = OrthogonalPolynomialFeatures(
+                self.degree,
+                [str(polynomial) for polynomial in self.polynomials_],
+                truncation=self.truncation,
+                weights=self.weights,
+                multiindices=self.multiindices_,
+            )
+
+            # create pipeline
+            self.pipeline_ = Pipeline([("basis", basis), ("solver", solver)])
+
+            # solve for coefficients
+            self.pipeline_.fit(X_scaled, y)
+
+            # for convenient access to multiindices, norms and coefficients
+            self.multiindices_ = self.pipeline_["basis"].multiindices_
+            self.norms_ = self.pipeline_["basis"].norms_
+            self.coef_ = self.pipeline_["solver"].coef_
+
+            # do not update multiindices in last iteration
+            if iter < max_iter - 1:
+                self.multiindices_ = self.strategy_.propose(self)
+
+        # by convention, fit returns itself
+        return self
+
+    def predict(self, X):
+        """Predict using this Polynomial Chaos model.
+
+        Parameters
+        ----------
+        X : array-like of shape (n_samples, n_features)
+            Points where the Polynomial Chaos expansion should be evaluated.
+
+        Returns
+        -------
+        y : ndarray of shape (n_samples,)
+            Polynomial Chaos predictions in query points.
+        """
+        # check if this estiamtor is fitted
+        check_is_fitted(self)
+
+        # check input features
+        X = self._validate_data(X, reset=False, ensure_2d=True)
+
+        # scale features
+        X_scaled = np.zeros_like(X)
+        for j, (distribution, polynomial) in enumerate(
+            zip(self.distributions_, self.polynomials_)
+        ):
+            X_scaled[:, j] = polynomial.scale_features_from_distribution(
+                X[:, j], distribution
+            )
+
+        y_fit = self.pipeline_.predict(X_scaled)
+        return y_fit * self.output_std_ + self.output_mean_
+
+    def joint_sens(self, *features):
+        r"""Return the joint sensivitivity indices.
+
+        Given a Polynomial Chaos expansion
+
+        .. math::
+            y = \sum_u c_u \Psi_u(\xi)
```

Comment:
How is `\Psi_u` defined here?

Reply:
$\Psi_u$ denotes the $u$ th orthogonal polynomial in the `OrthogonalPolynomialFeatures` basis. I've added a clarification on the meaning of $c_u$ and $\Psi_u$ in the docstrings of all relevant methods that use this definition.